ta/pkcs11/src/pkcs11_token.c

```diff
@@ -24,12 +24,8 @@
 #include "serializer.h"
 #include "token_capabilities.h"
 
-/* Provide 3 slots/tokens, ID is token index */
-#ifndef CFG_PKCS11_TA_TOKEN_COUNT
-#define TOKEN_COUNT		3
-#else
+/* Number of tokens implemented by the TA. Token ID is the token index */
 #define TOKEN_COUNT		CFG_PKCS11_TA_TOKEN_COUNT
```

Comment:
You could as well use `CFG_PKCS11_TA_TOKEN_COUNT` in the code below, but I'm fine with keeping this define if you prefer (it is indeed shorter / easier to read).

Reply:
agree. The macro is used at several places in this file. Easier to read with `TOKEN_COUNT`.